lib/dor/services/technical_metadata_service.rb

```diff
@@ -50,6 +50,8 @@ def self.test_jhove_service
     def self.get_content_group_diff(dor_item)
       inventory_diff = dor_item.get_content_diff('all')
       inventory_diff.group_difference('content')
+    rescue Dor::Exception # no contentMetadata
+      Moab::FileGroupDifference.new
     end
```

Comment:
`get_content_diff` throws Dor::Exception or Dor::ParameterError directly -- only the Dor::Exception case is of interest. The SdrClient could throw other exceptions but those probably should throw upwards.